amivapi/events/validation.py

```diff
@@ -63,6 +63,21 @@ def _validate_json_event_field(self, enabled, field, value):
             for error in validator.iter_errors(data):
                 self._error(field, error.message)
 
+    def _validate_not_blacklisted(self, enabled, field, user_id):
+        """Validate if the user is not on the blacklist.
+
+        The rule's arguments are validated against this schema:
+        {'type': 'boolean'}
+        """
+        if enabled:
+            count = current_app.data.driver.db['blacklist'].count({
```

Comment:
It seems that the `count` operation [is deprecated](https://docs.mongodb.com/manual/reference/method/cursor.count/) and `count_documents` [should be used instead](http://api.mongodb.com/python/current/api/pymongo/collection.html?highlight=count_documents#pymongo.collection.Collection.count_documents). It seems to have the same api as `count`, so this should be an easy change.